src/overlays/actors/ovl_Tools/ac_tools.h

```diff
@@ -14,4 +14,67 @@ typedef struct Tools {
     /* 0x000 */ Actor actor;
 } Tools; // size = 0x174
```

Comment:
Isn't this a bit unnecessary?  While I know that each profile has their own actor and what not, this is only referenced in the profile, and I think if it just has the actor member, maybe the profile should have `sizeof(Actor)` instead. But that's just my opinion

Reply:
Even if it isn't required, having a struct for this actor alread is nice because it allows expanding the actor with more members. We usually keep it this way even for actors that only have the actor struct

Reply:
Well, the tools, and even the functions in this actor use the base tools actor struct lol
But I understand

Reply:
I think I know what you mean by looking at the code of the actor now, it is pretty weird that the actor itself doesn't matter at all. the whole thing is more of an spawner of other actors